Wrappers/Python/cil/optimisation/operators/BlockOperator.py

```diff
@@ -63,139 +66,143 @@ def __init__(self, *args, **kwargs):
             :param: shape (:obj:`tuple`, optional): If shape is passed the Operators in 
                   vararg are considered input in a row-by-row fashion. 
                   Shape and number of Operators must match.
-                  
+
         Example:
             BlockOperator(op0,op1) results in a row block
             BlockOperator(op0,op1,shape=(1,2)) results in a column block
         '''
         self.operators = args
         shape = kwargs.get('shape', None)
         if shape is None:
-            shape = (len(args),1)
+            shape = (len(args), 1)
         self.shape = shape
-        n_elements = functools.reduce(lambda x,y: x*y, shape, 1)
+        n_elements = functools.reduce(lambda x, y: x*y, shape, 1)
         if len(args) != n_elements:
             raise ValueError(
-                    'Dimension and size do not match: expected {} got {}'
-                    .format(n_elements,len(args)))
+                'Dimension and size do not match: expected {} got {}'
+                .format(n_elements, len(args)))
         # TODO
         # until a decent way to check equality of Acquisition/Image geometries
-        # required to fullfil "Operators in a Block are required to have the same 
+        # required to fullfil "Operators in a Block are required to have the same
         # domain column-wise and the same range row-wise."
-        # let us just not check if column/row-wise compatible, which is actually 
+        # let us just not check if column/row-wise compatible, which is actually
         # the same achieved by the column_wise_compatible and row_wise_compatible methods.
-        
+
         # # test if operators are compatible
         # if not self.column_wise_compatible():
         #     raise ValueError('Operators in each column must have the same domain')
         # if not self.row_wise_compatible():
         #     raise ValueError('Operators in each row must have the same range')
```

Comment:
Looks like we need to open an issue about this and not just rely on this comment!!!

Reply:
Have opened issue #1543